reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+
+      metafields:
+        type: array
+        $ref: '#/definitions/metafield'
+
+      shipping_line:
+        type: object
+        $ref: '#/definitions/shipping_line'
+
+      tags:
+        type: string
+        description: Tags are additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. Each individual tag is limited to 40 characters in length.
+      tax_exempt:
+        type: boolean
+        description: Sets whether taxes are exempt for this draft order. If this value is `false`, Shopify will honor the `tax_exempt` value for each `line_item`.
+        enum:
+        - true
+        - false 
+      tax_lines:
+        type: array
+        description: An array of `tax_line` objects, each of which details the taxes applicable to the checkout. 
+        items:
+          $ref: '#/definitions/tax_line'
+        readOnly: true
+      discount:
+        type: object
+        $ref: '#/definitions/discount'
+        description: Order level discount.
+      taxes_included:
+        type: boolean
+        description: States whether or not taxes are included in the order subtotal.
+        readOnly: true
+        enum:
+        - true
+        - false
+
+      total_tax:
+        type: string
+        description: Total tax amount.
+        readOnly: true
+
+      subtotal_price:
+        type: string
+        description: Subtotal after discounts, before shipping and taxes have been applied.
+        readOnly: true
+
+      total_price:
+        type: string
+        description: Includes discounts, shipping and taxes.
+        readOnly: true
+
+      completed_at:
+        type: string
+        format: date-time
+        description: Date at which an order was created and the draft order moved to “completed” status.
+        readOnly: true
+      created_at:
+        type: string
+        format: date-time
+        description: By default, this auto-generated property is the date and time when the order was created in Shopify, in ISO 8601 format. If you are importing orders to the Shopify platform from another system, the writable `processed_at` property will override the `created_at date`.
+        readOnly: true
+
+      updated_at:
+        type: string
+        format: date-time
+        description: The date and time when the draft order was last updated.
+        readOnly: true
+
+      status:
+        type: string
+        enum:
+        - open
+        - invoice_sent
+        - completed
+        description: String describing the state of the draft order - `open`, `invoice sent`, `completed`.
+        readOnly: true
+
+  draft_order_invoice:
+    type: object
+    properties:
+      to:
+        type: string
+        description: >-
+          Email associated with the `draft_order_invoice`.  If not specified, the email associated with the draft order, or the customer's email will be used instead.  If no email address is specified an error is returned.
```

Comment:
If the draft order has no email address to use, then you must specify an email address, otherwise an error is returned.

Bad english, but the text right now implies an error exists all of the time.